docs/WindowsBuild.md

```diff
@@ -59,12 +60,17 @@ git clone https://github.com/apple/swift-compiler-rt compiler-rt
 git clone https://github.com/apple/swift
 git clone https://github.com/apple/swift-corelibs-libdispatch
 git clone https://github.com/apple/swift-corelibs-foundation
+git clone https://github.com/apple/swift-lldb lldb
 ```
 
 ### 3. Acquire ICU
 1. Download ICU from [ICU Project](http://site.icu-project.org) for Windows x64 and extract the binaries.
 1. Add the `bin64` folder to your `Path` environment variable.
 
+```cmd
```

Comment:
I think that we can sink this to the end.  The binaries are only needed when running the tests, not building.